website/showcase.json

```diff
@@ -295,7 +295,7 @@
       "icon": "baidu.png",
       "linkPlayStore": "https://play.google.com/store/apps/details?id=com.baidu.searchbox&hl=en&gl=US",
       "linkAppStore": "https://apps.apple.com/us/app/%E6%89%8B%E6%9C%BA%E7%99%BE%E5%BA%A6-%E7%99%BE%E5%BA%A6%E4%B8%80%E4%B8%8B%E4%BD%A0%E5%B0%B1%E5%BE%97%E5%88%B0/id382201985",
-      "infoLink": "http://baike.baidu.com/link?url=TW8YhcVN4tO_Jz5VqMclCjGhf12EEqMD_TeVC6efe2REZlx80r6T0dX96hdmNl36XogLyExXzrvFU9rFeqxg_K",
+      "infoLink": "https://baike.baidu.com/link?url=TW8YhcVN4tO_Jz5VqMclCjGhf12EEqMD_TeVC6efe2REZlx80r6T0dX96hdmNl36XogLyExXzrvFU9rFeqxg_K",
```

Comment:
@cortinico `infoLink` for Baidu seems to be broken. (_at least in India_ 🙂)
May you ping someone from the Baidu team on Discord to submit PR w/ the fix?

Reply:
Looks like `Baidu(dot)com` is blocked in India 🙂.

Reply:
Can confirm that link work for me. 🙂